keda/values.yaml

```diff
@@ -36,8 +36,16 @@ aadPodIdentity: ""
 # will be mounted to the /grpccerts path on the Pod
 grpcTLSCertsSecret: ""
 
+## Logging level for KEDA Controller 
```

Comment:
Can we somehow align the way we configure them/use them - WIth that I mean both use info or int?

Reply:
Unfortunately not in the moment, as Metrics Server internally uses https://github.com/kubernetes-sigs/custom-metrics-apiserver and we depends on it's way of logging (int based)

Reply:
Ok, no problem!